sklearn/tree/_export.py

```diff
@@ -243,7 +243,7 @@ def get_color(self, value):
         # compute the color as alpha against white
         color = [int(round(alpha * c + (1 - alpha) * 255, 0)) for c in color]
         # Return html color code in #RRGGBB format
-        return "#%2x%2x%2x" % tuple(color)
+        return "#%02x%02x%02x" % tuple(color)
```

Comment:
May you add a test to `sklearn/tree/tests/test_export.py` to check that this fixes the bug?